devel/cluster/export-logs.sh

```diff
@@ -22,9 +22,17 @@ SCRIPT_ROOT=$(dirname "${BASH_SOURCE}")
 export REPO_ROOT="${SCRIPT_ROOT}/.."
 source "${SCRIPT_ROOT}/../lib/lib.sh"
 
+IS_OPENSHIFT=${IS_OPENSHIFT:-"false"}
+
 # Require helm available on PATH
 check_tool kind
 
 LOGS_DIR="${ARTIFACTS:-$REPO_ROOT/_artifacts}/cert-manager-e2e-logs"
 rm -rf $LOGS_DIR && mkdir -p $LOGS_DIR
-kind export logs $LOGS_DIR --name "${KIND_CLUSTER_NAME}"
+if [[ "$IS_OPENSHIFT" == "true" ]] ; then
+  for container in $(docker ps -a | awk '{if(NR>1) print $NF}'); do
```

Comment:
I'm slightly confused by the `awk` here - can this be done with `docker ps -a -q`? not sure what `if(NR>1)` means though 😅

Reply:
i cannot remember exactly why i did that... so i'll change it!